packages/jest-mock/src/__tests__/index.test.ts

```diff
@@ -653,6 +663,16 @@ describe('moduleMocker', () => {
       expect(obj.func()).not.toBe('some text');
     });
 
+    it('supports mock value once returning undefined when not passing a value', () => {
+      const obj = {
+        func: () => 'some text',
+      };
+
+      moduleMocker.spyOn(obj, 'func').mockReturnValueOnce();
+
+      expect(obj.func()).toBeUndefined();
```

Comment:
Please add another check showing that second call of the method does not return undefined.